spec/factories/item_tags.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
```

Comment:
I don't think we need this. The joins table will automatically populate with the relationship in place.